src/applications/edu-benefits/10203/containers/IntroductionPage.jsx

```diff
@@ -279,19 +250,16 @@ export class IntroductionPage extends React.Component {
 }
 
 IntroductionPage.propTypes = {
-  apiVersion: PropTypes.object,
   getRemainingEntitlement: PropTypes.func,
   isLoggedIn: PropTypes.bool,
   remainingEntitlement: PropTypes.object,
   route: PropTypes.object,
-  useEvss: PropTypes.bool,
 };
 
 const mapStateToProps = state => {
   return {
     isLoggedIn: state.user.login.currentlyLoggedIn,
     remainingEntitlement: state.post911GIBStatus.remainingEntitlement,
-    useEvss: toggleValues(state)[FEATURE_FLAG_NAMES.stemSCOEmail],
```

Comment:
this feature flag entry was not present in the json file, so I went ahead and removed this prop